proto/panacea/datapool/v2/genesis.proto

```diff
@@ -25,4 +24,10 @@ message Params {
   ];
   uint64 data_pool_code_id = 2;
   string data_pool_nft_contract_address = 3;
+}
+
+// WhiteList define white list of data pool
+message WhiteList {
```

Comment:
IMHO, the word `WhiteList` is not proper for this case because we don't have an opposite concept `BlackList` in this case.
It would be enough to say `NFTTransferQueue` or so.

Reply:
```proto
message GenesisState {
   ...
   repeated DelayedNFTTransfer delayed_nft_transfes
}

message DelayedNFTTransfer {
   uint64_t pool_id
   string address
}
```
What do you think about this?

Reply:
Good. 
And depending on when and how to distribute this NFT, we don't need this kind of buyer lists.
For now, I'll rename it!

Reply:
Good.
I wish there was a similar name for the pool revenue distribute I'm developing